fuzz/trinity.py

```diff
@@ -40,9 +40,10 @@ def setUp(self):
         """
         Add not root user
         """
-        process.run('groupadd trinity', sudo=True)
+        process.run('groupadd trinity', sudo=True, ignore_status=True)
         process.run(
-            'useradd -g trinity  -m -d /home/trinity  trinity', sudo=True)
+            'useradd -g trinity  -m -d /home/trinity  trinity',
+            ignore_status=True, sudo=True)
```

Comment:
So, would you continue even if it fails ?
Also, is it not needed to set `ignore_status=True` to `usermod` in the next line ?